### ISSUE TITLE
Best settings for minimizing filesize and load on computer?

### ISSUE THREAD

**Issue by rabarrett8** [MaartenBaert/ssr#347]
Hi,

I like to record my opponent's turn in a turn-based game after I go to sleep.  To do that, I start screen recorder when I go to bed, turn off the monitor, and leave it running until I wake up and have time to check it again (8-10 hours later usually).  Right now, to minimize the file size, I set frame rate at 3 and put the Constant rate factor at 10.  I am also using Matroska and the H.264 Codec.  

Right now this works pretty well.  It is not an action game, but just shows someone moving units across a maps, usually fairly slowly without a lot of movement.  All I really need amounts to a photo every 1-2 seconds.  

Do you have any suggestions for better settings?
My first priority is to minimize file size so that I don't run out of disk space or have a giant file the next day (I might eventually leave it running for a day or two).  My second priority is to minimize the load on the computer so that it doesn't have to do any needless processing.

Thank you for any advice,

Richard

**Comment by Lucki** [comment]
You should take a look at the constant rate factor (crf) - it's really low with 10.
https://trac.ffmpeg.org/wiki/Encode/H.264#a1.ChooseaCRFvalue :
> The range of the quantizer scale is 0-51: where 0 is lossless, 23 is default, and 51 is worst possible.  A lower value is a higher quality and a subjectively sane range is 18-28. Consider 18 to be visually lossless or nearly so: it should look the same or nearly the same as the input but it isn't technically lossless.

Higher crf means less file size ('cause of the less detailed data), but be careful with too high values.
Decide whether you want to be lossless (crf=0), or not (crf=[18-28]).

Second optimization would be a slower preset - the cpu has more time to compress your video, but this doesn't take too much.
https://trac.ffmpeg.org/wiki/Encode/H.264#a2.Chooseapreset :
> The general guideline is to use the slowest preset that you have patience for. Current presets in descending order of speed are: `ultrafast`, `superfast`, `veryfast`, `faster`, `fast`, `medium`, `slow`, `slower`, `veryslow`, [`placebo`].

Experiment with which preset you can capture fluid. This obviusly don't get too crazy at 3 FPS.

**Comment by MaartenBaert** [comment]
Correct. Also, you might want to consider scaling the video down if possible. But just increasing the crf value to 23 (the default) should already make a huge difference.
